lib/compiler/src/beam_ssa_codegen.erl

```diff
@@ -1450,6 +1450,10 @@ cg_copy_1([], _St) -> [].
                           element(1, Val) =:= atom orelse
                           element(1, Val) =:= literal)).
 
+bif_to_test(min, Args, Fail, St) ->
+    bif_to_test('and', Args, Fail, St);
```

Comment:
Perhaps add a comment saying that `min` and `max` are only rewritten as tests (`z0/_3 = bif:max x0/Y, x0/Y`) when both arguments are known to be booleans?